tango_simlib/tests/DishElementMaster_SIMDD.json

```diff
@@ -190,6 +190,70 @@
           "quantity_simulation_type": "ConstantQuantity"
         }
       }
+    },
+    {
+      "basicAttributeData": {
```

Comment:
This also applies to the basicCommandData, much of that information is duplicated. However in this case it is because the fields are set out to be mandatory (as specified in the SIMMD schema). A ticket has been added to the backlog to update the schema file [LMC-285](https://skaafrica.atlassian.net/browse/LMC-285) to address the duplication of information.